test/mocha-opts

```diff
@@ -1,3 +0,0 @@
---require test/common.js
```

Comment:
@lennym  did you intend to delete these?

Reply:
Yes. There were two files with identical content: `mocha-opts` and `mocha.opts` of which the former (this one) didn't do anything. So I deleted it.